src/components/Node/Node.tsx

```diff
@@ -89,6 +89,11 @@ const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = memo(
       const isAccepted = wasActionTaken
         ? documentation?.resourceType !== 'DocumentReference'
         : null;
+      if (isAccepted === false) {
+        topLevelClasses.push(styles.declined);
+        if (documentation) documentation.status = 'Declined';
```

Comment:
Is there a way to do this without changing the status. I don't _think_ we are writing this back anywhere but just in case we should avoid it. 'Declined' is not actually a status for the resources we are using for documentation. 
The two solutions I can think of are
1. Passing the `isAccepted` flag down to the `ExpandedNode` (as a prop)
2. A decline is done through a DocumentReference with an attached note. You can check the status of that note for accepted/declined

I think the second method is going to be too much hassle so I would go with the first.

Reply:
Yeah I think passing the flag down to the ExpandedNode makes more sense.

Reply:
Got it, thank you for the feedback! Will modify and re-push.